### PR TITLE
Update syn and quote

### PR DESCRIPTION
**Pull request by Eijebong**

### CODE REVIEW

File: gumdrop_derive/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "gumdrop_derive"
-version = "0.4.0"
```
Comment (murarth) [inline gumdrop_derive/Cargo.toml]: I prefer to do changes to the project's version as a standalone commit. Revert this line and I'll do a release commit, then publish after merging.

Comment (Eijebong) [reply gumdrop_derive/Cargo.toml]: Sure :)

**Comment by murarth** [comment]
Published 0.4.1 of `gumdrop_derive`. Thanks for the contribution.

**Comment by Eijebong** [comment]
Thanks :)

**Pull request merged**